src/auto-trace.helper.js

```diff
@@ -33,6 +32,7 @@ export function wrapObjectWithError(err, stacktraceErr, extraContext) {
 			console.warn('auto-trace: You are trying to throw something that cannot be stringified', ex);
 			errOut.message = err;
 		}
+		errOut = removeAutoTraceFromErrorStack(errOut);
```

Comment:
When err.stack is accessed before err.message is set, the stack property will be computed without the err.message string. This fix sets the message before removing auto-trace from the stack.